modules/installation/proc-rhdh-deploy-gke-helm.adoc

```diff
@@ -0,0 +1,181 @@
++
+--
+[source,yaml,subs="attributes+"]
+----
+apiVersion: networking.gke.io/v1
+kind: ManagedCertificate
+metadata:
+  name: <rhdh_certificate_name>
+spec:
+  domains:
+    - <rhdh_domain_name>
+----
+--
+For more information about setting up a Google-managed certificate, see https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs?hl=en#setting_up_a_google-managed_certificate[Setting up a Google-managed certificate].
+
+. Create a `FrontendConfig` object to set a policy for redirecting to HTTPS. You will attach this policy to the Ingress. 
++
+--
+[source,yaml,subs="attributes+"]
+----
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: <ingress_security_config>
+spec:
+  sslPolicy: gke-ingress-ssl-policy-https
+  redirectToHttps:
+    enabled: true
+----
+--
+For more information about setting a policy to redirect to HTTPS, see https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration?hl=en#https_redirect[HTTP to HTTPS redirects].
+
+. Create a file named `values.yaml` using the following template:
++
+[source,yaml,subs="attributes+"]
+----
+global:
+  host: <rhdh_domain_name>
+route:
+  enabled: false
+upstream:
+  service:
+    type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: gce
+      kubernetes.io/ingress.global-static-ip-name: <ADDRESS_NAME>
+      networking.gke.io/managed-certificates: <rhdh_certificate_name>
+      networking.gke.io/v1beta1.FrontendConfig: <ingress_security_config>
+    className: gce
+  backstage:
+    image:
+      pullSecrets:
+      - rhdh-pull-secret
+    podSecurityContext:
+      fsGroup: 2000
+  postgresql:
+    image:
+      pullSecrets:
+      - rhdh-pull-secret
+    primary:
+      podSecurityContext:
+        enabled: true
+        fsGroup: 3000
+  volumePermissions:
+    enabled: true
+----
+. Run the following command in your terminal to deploy {product-short} using the latest version of Helm Chart and using the values.yaml file created in the previous step:
++
+[source,terminal,subs="attributes+"]
+----
+helm -n <your_namespace> install -f values.yaml <your_deploy_name> \
+  openshift-helm-charts/redhat-developer-hub \
+  --version {product-chart-version}
+----
++
+For the latest Helm Chart version, see https://github.com/openshift-helm-charts/charts/tree/main/charts/redhat/redhat/redhat-developer-hub
++
+. Confirm that the deployment is complete.
++
+[source,terminal,subs="attributes+"]
+----
+kubectl get deploy <you_deploy_name>-developer-hub -n <your_namespace>
+----
+
+. Verify that the service and ingress were created.
++
+[source,terminal,subs="attributes+"]
+----
+kubectl get service -n <your_namespace>
+kubectl get ingress -n <your_namespace>
+----
++
+[NOTE]
+Wait for the `ManagedCertificate` to be provisioned. This can take a couple of hours.
+
+. Access RHDH with `https://<rhdh_domain_name>`
++
+[IMPORTANT]
+Use the HTTPS protocol, not HTTP.
```

Comment:
We actually don't need this note for GKE, because we are setting a redirect from HTTP to HTTPS. My bad - I forgot this in the draft.
```suggestion
```